app/src/main/java/com/android/sample/model/image/ImageRepository.kt

```diff
@@ -34,4 +34,6 @@ interface ImageRepository {
       onSuccess: (List<Bitmap>) -> Unit,
       onFailure: (Exception) -> Unit
   )
```

Comment:
Added a new method deleteProfilePicture to delete a profile picture from Firebase storage.